tests/chainer_tests/functions_tests/array_tests/test_resize_images.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+
+import numpy
+
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+@testing.parameterize(*testing.product({
+    'in_shape': [(2, 3, 8, 6), (2, 1, 4, 6)],
+}))
```

Comment:
Are there any reasons to limit the tests to `numpy.float32`?
It seems that your proposed function also accepts `float64` and `float16`.